doc/changes/whats_new.rst

```diff
@@ -0,0 +1,24 @@
+.. _whats_new:
+
+What's new
+==========
+
+.. currentmodule:: benchopt
+
+
+.. include:: _1_5_0.rst
+
+.. include:: _1_4_0.rst
+
+.. include:: _1_3_1.rst
+
+.. include:: _1_3_0.rst
+
+.. include:: _1_2_0.rst
+
+.. include:: _1_1_0.rst
+
+.. include:: _1_0_0.rst
+
+
+.. include:: ../names.inc
```

Comment:
This looks more complex than what we had before, are we sure we want to bother with this?

Reply:
With more releases, the ``whats_new`` file has become huge and hard to handle.
The split makes it more manageable, right?

FYI: I don't have a strong opinion about this change

Reply:
I'm in favor of addressing it if and only if it becomes a problem. In this PR it creates more files instead of a single renaming, thus makes reviewing harder than it could.